ReactiveCocoa/UIKit/UITextField.swift

```diff
@@ -9,12 +9,17 @@ extension Reactive where Base: UITextField {
 	}
 
 	/// A signal of text values emitted by the text field upon end of editing.
+	///
+	/// - note: To observe text values that change on all editing events,
```

Comment:
Perhaps just _text values being emitted upon any changes_? IIRC it does not emit values upon end of editing, for example.